src/main/java/com/joejoe2/chat/exception/ChannelDoesNotExist.java

```diff
@@ -1,7 +1,7 @@
 package com.joejoe2.chat.exception;
 
 public class ChannelDoesNotExist extends Exception {
-  public ChannelDoesNotExist(String message) {
-    super(message);
+  public ChannelDoesNotExist(String channelId) {
+    super("channel %s does not exist !".formatted(channelId));
```

Comment:
It's not a good practice to hardcode an exception message in the exception itself. It should be passed from the class where it's being thrown.

Reply:
I think a builder method like ofChannalIf(strimg id) to have this template msg will be better. And we can leave the constructor to accept custom  message.